src/tests/ClockReduction/clock_removal_test.rs

```diff
@@ -0,0 +1,28 @@
+#[cfg(test)]
+pub mod clock_removal_tests {
+    use crate::tests::ClockReduction::helper::test::assert_edges_in_component;
+    use crate::DataReader::json_reader::read_json_component;
+    use std::collections::HashSet;
+
+    // Tests that the clocks that are never never used in any guards are removed.
```

Comment:
```suggestion
    // Tests that the clocks that are never used in any guards are removed.
```
I assume the double negative never never was a mistake

Reply:
@pberte20